### PR TITLE
Update about team data

### PR DESCRIPTION
**Pull request by Polarpi**
* update co-directors + message from arpan
* update community + external directors
* add academy + PR leads
* update PMs + tech leads
* move graduated members to alumni section

### CODE REVIEW

**Review by arpanlaha — APPROVED** [review]
🍪

File: data/teamData.js [hunk 1]
```diff
@@ -116,19 +105,19 @@ export default {
       linkedin: "https://www.linkedin.com/in/anisharao14/"
     },
     {
-      name: "Aryn Harmon",
+      name: "Neeraj Aggarwal",
       role: "Software Developer",
```
Comment (rgychiu) [inline data/teamData.js:109]: I think this should be changed to reflect his most recent position

File: data/teamData.js [hunk 2]
```diff
@@ -116,19 +105,19 @@ export default {
...
-      linkedin: "https://www.linkedin.com/in/aryn/"
+      linkedin: "https://www.linkedin.com/in/neeraj-aggarwal/"
     },
     {
-      name: "Daniel Choi",
+      name: "Jacqueline Osborn",
       role: "Software Developer",
```
Comment (rgychiu) [inline data/teamData.js:114]: Same here for Jackie

**Review by rgychiu — CHANGES_REQUESTED** [review]

File: data/teamData.js (hunk 1 above)
Comment (Polarpi) [reply data/teamData.js:109]: I think this is correct? I updated everything for the fall

File: data/teamData.js (hunk 2 above)
Comment (Polarpi) [reply data/teamData.js:114]: This is also updated for the fall sem

**Review by rgychiu — APPROVED** [review]

**Pull request merged**